getsmarter_api_clients/geag.py

```diff
@@ -23,8 +25,101 @@ def get_terms_and_policies(self):
         response.raise_for_status()
         return response.json()
 
-    def create_allocation(self):
+    def create_allocation(
+        self,
+        payment_reference,
+        address_line1,
+        city,
+        postal_code,
+        state,
+        state_code,
+        country,
+        country_code,
+        first_name,
+        last_name,
+        email,
+        date_of_birth,
+        terms_accepted_at,
+        currency,
+        order_items,
+        address_line2=None,
+        mobile_phone=None,
+        work_experience=None,
+        education_highest_level=None
+    ):
         """
         Create an allocation (enrollment) through GEAG.
+
+        Args:
+            payment_reference (str): Reference used by enterprise partner
+            when payment is made to GetSmarter
+            address_line1 (str): Address Line 1
+            city (str): City
+            postal_code (str): Postal code
+            state: State
+            state_code: State code
+            country (str): Country
+            country_code (str): Country code,
+            first_name (str): First name
+            last_name (str): Last name
+            email (str): Email
+            date_of_birth (str): Date of birth
+            terms_accepted_at (str): ISO 8601 timestamp of
+            when the terms and policies were accepted
+            currency: One of ['USD', 'GBP', 'ZAR', 'EUR', 'AED',
+            'SGD', 'HKD', 'SAR', 'INR', 'CAD']
+            order_items: Items ordered
+            address_line2: Adress Line 2
+            mobile_phone: Mobile phone number
+            work_experience: One of ['None', '1 to 5 years',
+            '5 to 15 years', 'More than 15 years']
+            education_highest_level: Highest education reached by the learner
```

Comment:
[curious] Does this field only accept certain values, like `work_experience` above? If so, is it worth including them here, too?